engine/lib/navigation.php

```diff
@@ -154,7 +158,7 @@ function elgg_unregister_menu_item($menu_name, $item_name) {
  * @since 1.8.0
  */
 function elgg_is_menu_item_registered($menu_name, $item_name) {
-	global $CONFIG;
+	$CONFIG = _elgg_services()->config->getStorageObject();
```

Comment:
why not use `$menus = elgg_get_config('menus');` like you do in the other functions in this file?